content/OTP/Guides/OTP_Walk-through.adoc

```diff
@@ -1,16 +1,17 @@
 == One-Time Password (OTP) Walk-Through
 
-We are going to  integrating the Yubico One-Time Password (OTP) protocol with your application or framework, step by step.
+
+We are going to walk-through integrating the Yubico One-Time Password (OTP) protocol with your application or framework, step by step.
 
 TIP: This walk-through is designed for people who prefer *learning by doing*. If you prefer to learn concepts from the ground up, check out our link:https://www.yubico.com/wp-content/uploads/2012/10/YubiKey-Authentication-Module-Design-Guideline-v1.0.pdf[YubiKey Authentication Module Design Guideline]. You should find this and the guide are complementary to each other.
 
-The  is divided into several sections:
+The walk-through is divided into several sections:
 
 * Overview provides the fundamentals of OTP
 * Setup provides you With a starting point to follow the walk-through
-* Enable your Yubikey API authenticator
-* Create your verification client
```

Comment:
Chris Halos: I see multiple instances of Verification Server - shouldn't this always be Validation Server?